client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -13,6 +13,7 @@ class ElementEditor extends PureComponent {
       <div className="element-editor">
         <ToolbarComponent />
         <ListComponent pageId={pageId} />
+        <HeaderComponent/>
```

Comment:
The HeaderComponent should be added to the Element, rather than this part, right?